examples/README.md

```diff
@@ -1,13 +1,59 @@
 # Example: Provision an SDDC
```

Comment:
with all the workflows listed below, does this heading still apply?

Reply:
Good catch. I will change it

Reply:
Done